bigchaindb/common/schema/transaction.yaml

```diff
@@ -88,11 +88,10 @@ definitions:
         Type of the transaction:
 
         A ``CREATE`` transaction creates an asset in BigchainDB. This
-        transaction has outputs (conditions) but no inputs (fulfillments),
-        so a dummy fulfillment is used.
+        transaction has outputs but no inputs, so a dummy input is created.
 
         A ``TRANSFER`` transaction transfers ownership of an asset, by providing
-        fulfillments to conditions of earlier transactions.
+        fulfillments to outputs conditions of earlier transactions.
```

Comment:
The wording here is a bit odd. How about:

fulfillments to conditions in outputs of earlier transactions.